CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 ## HEAD
 
+## 0.16.7
+###### _Jan 15, 2017_
+
+A big thanks for the 20 contributors that are making that release possible.
```

Comment:
'A big thanks to the 20 contributors who are making this release possible.'

(Or possibly ''A big thanks to the 20 contributors who made this release possible.' - for anyone reading the changelog the release is a past event.)